internal/lib/remove.go

```diff
@@ -20,17 +20,22 @@ func (c *ContainerServer) Remove(ctx context.Context, container string, force bo
 	cStatus := ctr.State()
 	switch cStatus.Status {
 	case oci.ContainerStatePaused:
-		return "", errors.Errorf("cannot remove paused container %s", ctrID)
+		if !force {
+			return "", errors.Errorf("cannot remove paused container %s", ctrID)
+		}
+		if err = c.runtime.UnpauseContainer(ctx, ctr); err != nil {
+			return "", errors.Wrapf(err, "unable to unpause container %s", ctrID)
+		}
 	case oci.ContainerStateCreated, oci.ContainerStateRunning:
-		if force {
-			if err = c.StopContainer(ctx, ctr, 10); err != nil {
-				return "", errors.Wrapf(err, "unable to stop container %s", ctrID)
-			}
-		} else {
+		if !force {
 			return "", errors.Errorf("cannot remove running container %s", ctrID)
 		}
 	}
 
+	if err = c.StopContainer(ctx, ctr, 10); err != nil {
+		return "", errors.Wrapf(err, "unable to stop container %s", ctrID)
+	}
+
```

Comment:
this StopContainer looks like it should be inside of the block above

Reply:
fixde